includes/nets-easy-functions.php

```diff
@@ -427,6 +427,10 @@ function nets_easy_all_payment_method_ids() {
  * @return string The payment method title.
  */
 function nexi_get_payment_method_title( $order, $method, $type ) {
+	if ( apply_filters( 'nexi_custom_payment_method_title', false ) ) {
+		return apply_filters( 'nexi_custom_payment_method_title', $order, $method, $type );
+	}
+
```

Comment:
This is not quite how a filter is supposed to work. The first parameter after a filter key is supposed to be the value that we want to use, and then any other filter should be the parameters used to decide on the value.

So in this case, just use the filter on the value we return normally on line 444, and filter that.
`return apply_filters( 'nexi_custom_payment_method_title', "$gateway / $method $type", $order, $method, $type) ;`
And remove this if statement.